modules/social_features/social_email_broadcast/social_email_broadcast.routing.yml

```diff
@@ -0,0 +1,7 @@
+social_email_broadcast.user_edit_page:
```

Comment:
We already have a route with the same purpose: `social_user.my_settings`, so why do we need to duplicate it here?

Reply:
The difference is in the path itself: `/edit-email-settings` versus `/my-settings`
But it's unimportant, so I removed the route.